rasa/nlu/test.py

```diff
@@ -352,11 +352,10 @@ def substitute_labels(labels, old, new):
     return [new if label == old else label for label in labels]
 
 
-def evaluate_entities(entity_results, interpreter, report_folder):  # pragma: no cover
+def evaluate_entities(entity_results, extractors, report_folder):  # pragma: no cover
```

Comment:
As you touched this file, would you mind adding types? We are cleaning up things on the way. Would be great if you could add types for all the methods you touched.
E.g. update the method definition to
``` 
def evaluate_entities(entity_results: List[EntityEvaluationResult], extractors: Set[Text], report_folder: Optional[Text]):
```

Reply:
Sure. Will do.

Reply:
Ok, it's ready for review.